Directory.Build.props

```diff
@@ -23,6 +23,10 @@
       Project="$(_OutputPath)MonoInfo.props"
       Condition="Exists('$(_OutputPath)MonoInfo.props')"
   />
+  <Import
+      Project="$(_OutputPath)XamarinAndroid.props"
```

Comment:
What filename is actually intended?  The message for this PR and xamarin/xamarin-android#5444 both mention `XAConfig.props`, *not* `XamarinAndroid.props`.  Should this be an `<Import/>` for `XAConfig.props` and *not* `XamarinAndroid.props`?  Or should the commit message and xamarin/xamarin-android#5444 be updated to mention `XamarinAndroid.props`?

Which filename is actually intended?

Reply:
Or is this for something completely different?

Reply:
good catch, that remained here from older versions of the change, I will remove it.